tests/wicked/advanced_ref.pm

```diff
@@ -33,22 +33,31 @@ sub create_tunnel_with_commands {
 
 sub run {
     my ($self) = @_;
+
     record_info('Test 1', 'Create a gre interface with IP commands');
-    $self->create_tunnel_with_commands("gre1", "24");
+    $self->create_tunnel_with_commands("gre1", "gre", "24");
     # Lock until parent creates mutex 'test_1_ready'
     mutex_lock('test_1_ready');
     # Unlock mutex to end test
     mutex_unlock('test_1_ready');
     assert_script_run("ifdown gre1");
 
     record_info('Test 3', 'Create a SIT interface with IP commands');
-    $self->create_tunnel_with_commands("sit1", "127");
-    # Lock until parent creates mutex 'test_1_ready'
+    $self->create_tunnel_with_commands("sit1", "sit", "127");
+    # Lock until parent creates mutex 'test_3_ready'
     mutex_lock('test_3_ready');
     # Unlock mutex to end test
     mutex_unlock('test_3_ready');
 
+    record_info('Test 6', 'Create a SIT interface with IP commands');
```

Comment:
why 6? there is another PR with 4th but did I missed 5th somewhere?

Reply:
I am trying to keep the "legacy ifcfg"-type tests in even numbers, so the xml cases go next. Example, test4 - bridge with ifcfg. Test5: bridge with xml.  and so on.